src/utilities/zAuctionRouting.ts

```diff
@@ -1,21 +1,35 @@
 import * as zAuction from "@zero-tech/zauction-sdk";
-import { zAuctionRoute } from "../types";
+import { zAuctionRoute, RouteUriToInstance } from "../types";
+import { Config } from "..";
 
 type GetDomainNameFromIdFunction = (domainId: string) => Promise<string>;
 
+// const zAuctionRouteUriToInstance: RouteUriToInstance = {};
```

Comment:
remove commented code here and elsewhere - there's a couple comments left in the tests